kernel-include/net/xia_ether.h

```diff
@@ -0,0 +1,30 @@
+#ifndef _NET_XIA_ETHER_H
+#define _NET_XIA_ETHER_H
+
+#include <linux/netdevice.h>
+#include <linux/netlink.h>
+
+/* Ethernet Principal */
+#define XIDTYPE_ETHER (__cpu_to_be32(0x22))
+
+struct rtnl_xia_ether_addrs {
+	__u16 attr_len;
+	__u8  interface_addr_len;
+	__u8  interface_addr[MAX_ADDR_LEN];
+	int   interface_index;
+};
+
+static inline int RTHA_OK(struct rtnl_xia_ether_addrs *rtha, int len)
+{
+	return len >= 0 && (unsigned)len >= sizeof(*rtha) &&
+		rtha->attr_len <= (unsigned)len;
+}
+
+static inline struct rtnl_xia_ether_addrs *RTHA_NEXT(
+	struct rtnl_xia_ether_addrs *rtha)
+{
+	return	(struct rtnl_xia_ether_addrs *)
+		(((char *)rtha) + NLMSG_ALIGN(rtha->attr_len));
+}
+#endif		/* _NET_XIA_ETHER_H */
+
```

Comment:
Drop this blank line.

Reply:
The error @AltraMayor pointed out in XIA-for-Linux initial pull request was the same one. As soon as i added a blank at its end that error was removed. So i think we should keep a blank line at the end of each file. (more info:https://stackoverflow.com/questions/34271528/what-is-little-red-arrow-and-no-sign-in-github-desktop)